src/lib/rule-finder.js

```diff
@@ -108,6 +108,6 @@ function RuleFinder(specifiedFile, options) {
   this.getUnusedRules = () => getSortedRules(unusedRules);
 }
 
-module.exports = function (specifiedFile, options = {}) {
+module.exports = function (specifiedFile, options) {
```

Comment:
Yeah, that's what I did at first, then switched it up so that the default was closer to where it was needed.  I'm happy to go either way as long as you both agree on the direction.

In fact, you can revert my last commit to get the default on the exported function because that's how I did it at first.